packages/fe-fpm-writer/src/view/index.ts

```diff
@@ -0,0 +1,78 @@
+import { create as createStorage } from 'mem-fs';
+import type { Editor } from 'mem-fs-editor';
+import { create } from 'mem-fs-editor';
+import type { CustomView, InternalCustomView } from './types';
+import { join } from 'path';
+import { render } from 'ejs';
+import { validateVersion, validateBasePath } from '../common/validate';
+import type { Manifest } from '../common/types';
+import { setCommonDefaults, getDefaultFragmentContent } from '../common/defaults';
+
+/**
+ * Enhances the provided custom view configuration with additional data.
```

Comment:
...with default data
(it's called like this with the other extensions and I think this fits better)

Reply:
renamed to `default data`